install/ansible/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM alpine:3.5
 MAINTAINER contiv
 
-RUN apk add --no-cache python-dev py-pip gcc musl-dev openssl-dev libffi-dev && \
-    pip install --upgrade pip && pip install ansible==2.2.0.0
+RUN DEV_PACKAGES="python-dev py-pip gcc musl-dev openssl-dev libffi-dev" \
+ && apk add --no-cache python openssl libffi $DEV_PACKAGES \
+ && pip install --upgrade pip \
+ && pip install ansible==2.2.0.0 \
```

Comment:
@unclejack, is our ansible swarm setup works fine with 2.2.1.0, we can update this as well.